evap/results/templates/results_index.html

```diff
@@ -19,8 +19,16 @@ <h5 class="card-title">
                 </span>
             </h5>
             <div class="row">
-                <div class="col-12 col-lg">
-                    <input type="search" name="search" class="form-control mb-3" placeholder="{% translate 'Search...' %}" />
+                <div class="col-12 col-lg mb-3 mb-lg-0">
+                    <label>
+                        <input type="search" name="search" class="form-control mb-3"
+                               placeholder="{% translate 'Search...' %}" />
```

Comment:
For consistency within this file, I wouldn't break this line.

```suggestion
                        <input type="search" name="search" class="form-control mb-3" placeholder="{% translate 'Search...' %}" />
```